src/server/api/routers/user.ts

```diff
@@ -6,15 +6,15 @@ import {
   publicProcedure,
 } from "~/server/api/trpc";
 
-import { prismaPostgres as prisma } from '~/server/db';
+import { TRPCError } from "@trpc/server";
 import { hashPassword } from "~/server/auth";
-import bcrypt from 'bcrypt'
+import { prismaPostgres as prisma } from "~/server/db";
 
 const userObject = z.object({
```

Comment:
We may want to narrow the validation here to match what we're doing clientside (have email be properly formatted instead of just string etc).

Reply:
yeap sounds good shall i do that in another pr?